de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/VirSatJettyServer.java

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.server;
+
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import static org.eclipse.jetty.servlet.ServletContextHandler.NO_SESSIONS;
+
+/**
+ * This class represents a Jetty Instance to run Virtual Satellite
+ * Stand alone as server application. The class runs some Jersey Servelets
+ * to provide a REST API to the outside world
+ *
+ */
+public class VirSatJettyServer {
+
+	public VirSatJettyServer() {
+	}
+	
+	private static final int VIRSAT_JETTY_PORT = 8000; 
+	
+	/**
+	 * Main entry point for Virtual Satellite Jetty Server
+	 * @param args Not used currently
+	 */
+	public static void main(String[] args) {
+		try {
+			new VirSatJettyServer().start().join();
+		} catch (Exception ex) {
+			ex.printStackTrace();
+		}
+	}
+
+	private Server server;
+	
+	/**
+	 *  Call this method to start the Virtual Satellite specific Jetty Server.
+	 *  This method blocks in a join statement to wait make the cally thread wait

```

Comment:
I think the first "want" is too much - the sentence sounds gramatically wrong. And "cally"?

Reply:
True... fixed...